Pollo/Views/PollBuilder/OnboardingView.swift

```diff
@@ -0,0 +1,524 @@
+//
+//  OnboardingView.swift
+//  Pollo
+//
+//  Created by Kevin Chan on 10/26/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+enum OnboardingStage {
+    case welcome
+    case createQuestion
+    case autofillChoices
+    case quizMode
+    case saveDraft
+    case startQuestion
+    case finished
+}
+
+class OnboardingView: UIView {
+
+    // MARK: - View vars
+    private let addOptionLabel = UILabel()
+    private let addOptionView = UIView()
+    private let animatedLabel = UILabel()
+    private let askQuestionBorder = UIView()
+    private let askQuestionLabel = UILabel()
+    private let buttonLabel = UILabel()
+    private let buttonView = UIView()
+    private let continueLabel = UILabel()
+    private let firstCorrectView = UIView()
+    private let firstOptionLabel = UILabel()
+    private let firstOptionView = UIView()
+    private let plusLabel = UILabel()
+    private let secondCorrectView = UIView()
+    private let secondOptionLabel = UILabel()
+    private let secondOptionView = UIView()
+    private let transitionButton = UIButton()
+    private let welcomeContainerView = UIView()
+    private let welcomeDescriptionLabel = UILabel()
+    private let welcomeTitleLabel = UILabel()
+    
+    // MARK: - Data vars
+    var containerFrame: CGRect!
+    var isTransitioning = false
```

Comment:
Is `isTransitiong` to prevent skipping a stage if a user taps during an animation? If so, it doesn't seem to be used anywhere.

Reply:
Had a separate variable `shouldTransition` used for this purpose, but I'll change that one to `isTransitioning` because that makes more sense. Thanks for catching that!